frontend/src/views/RouteFormView.vue

```diff
@@ -176,11 +194,18 @@ export default {
     },
     handleSubmit() {
       this.validateForm();
+      this.confirmationOffline = false;
+      this.confirmationOnline = false;
 
       if (!Object.keys(this.errors).length) {
         this[actions.SUBMIT]({ form: { ...this.route, syncId: Math.floor(Date.now() / 1000) } });
         this.route = { ...defaultFormState };
         this.selectedPassengers = [];
+        if (this.isOnline) {
+          this.confirmationOnline = true;
```

Comment:
I feel like we should consider scenario, when user fills in form twice in a row. Notification won't go away by itself ever with current strategy. I am amost tempted to suggest dismissing them with a timeout, but I am not sure if that is the best thing to do. What do you think?

Reply:
Notification will go away as it is reset in lines 197 and 198 - only one alert is shown at once.

I sometimes use "dismiss after a while" strategy with alerts but only with success-type ones and never with error-type. I'm not quite sure where to put this "success-but-not-yet-really" type, when app is offline. Maybe if we are going to have some notification service working through the entire app we could elaborate a consistent behaviour.

Reply:
I like the idea of centralized notifications system perhaps something like Snackbar component from material design? That would be great for phone usability and we could connect all user info to this, including sync messages, logout, etc.

https://vuematerial.io/components/snackbar
Lets discusson Wednesday!